python_scripts/repeatable/update_refids.py

```diff
@@ -62,7 +64,7 @@ def main(csv_path, dry_run=False):
                     print(post_response)
 
 
-# Call with `python update_agentids.py <filename>.csv resources`
+# Call with `python update_refids.py <filpath>.csv`
```

Comment:
Tiny tweak would be helpful here - should also include an example jsonl path since the script now will choke without it.